### PR TITLE
[CELEBORN-927][DOC] Guideline to add new RPC messages

### PR DESCRIPTION
**Pull request by waitinfuture**
### What changes were proposed in this pull request?
As title.

### Why are the changes needed?
As title.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Passes GA.

### CODE REVIEW

**Comment by waitinfuture** [comment]
cc @SteNicholas

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,3 +33,8 @@ There are already some further improvements on the schedule and welcome to conta
 8. Enhanced K8S support.
 9. Support spilled data.
 10. Locality awareness.
+
+## Guidelines
+### Adding RPC Messages
+We are in the process of replacing case class wrappers for RPC messages with raw PB. If you are to add new
```
Comment (SteNicholas) [inline CONTRIBUTING.md:39]:
```
When you adds new RPC message, it's recommended to follow raw PB message case, for example `RegisterWorker` and `RegisterWorkerResponse`. The RPC messages will be unified into raw PB messages eventually. 
```

Comment (waitinfuture) [reply CONTRIBUTING.md]: done, thanks

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request closed without merging**